### DISCUSSION TITLE
Manual/UIE-USS-PropertyTypes

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#545] [Notes]
# Manual/UIE-USS-PropertyTypes

https://docs.unity3d.com/Manual/UIE-USS-PropertyTypes.html

**Comment by vertxxyz** [comment]
The information on this page mostly relates to how to read the [USS common properties](https://docs.unity3d.com/Manual/UIE-USS-SupportedProperties.html) documentation, where you apply the listed syntax rules to formulate a valid property.
